src/risclog/sqlalchemy/tests/test_db.py

```diff
@@ -119,26 +119,35 @@ class Model_1(Base_1):
 
     class ObjectBase_2(ObjectBase):
         _engine_name = 'db2'
-    declarative_base(ObjectBase_2)
+    Base_2 = declarative_base(ObjectBase_2)
+
+    class Model_2(Base_2):
+        id = Column(Integer, primary_key=True)
 
     def tearDown():
         risclog.sqlalchemy.db.unregister_class(Model_1)
     request.addfinalizer(tearDown)
 
     database_1.create_all('db1')
+    database_2.create_all('db2')
 
     db = get_database(testing=True)
     # It used to be necessary to bind the session to the database to be
     # queried if more than one database was being used. This is no longer the
     # case as of SQLAlchemy 1.0.
     assert db.session.query(Model_1).count() == 0
 
+    Model_1.create().persist()
+    Model_2.create().persist()
+    import transaction
```

Comment:
Is there a reason, why we cannot move this import to the top? As this is currently not specified in the `setup.py`, it would be also good to have it there. As it is also used in `db.py` it can go to `install_requires`.